### PR TITLE
x264: bump to 20170521

### PR DESCRIPTION
**Pull request by kenmays**
**x264** is a free software library and application for encoding video streams into the H.264/MPEG-4 AVC compression format.
https://www.videolan.org/developers/x264.html

- Built and packaged successfully on hrev53121 x86. No issues.

### CODE REVIEW

**Comment by Begasus** [comment]
Did you check this with the recipes requiring lib:libx264?

**Comment by kenmays** [comment]
@Begasus - Yes. The API didn't change.

File: media-libs/x264/x264-20170521.recipe [hunk 1]
```diff
@@ -1,11 +1,11 @@
 SUMMARY="An encoder for x264 h.264/AVC"
 DESCRIPTION="x264 is a h.264/AVC encoder library."
 HOMEPAGE="https://www.videolan.org/developers/x264.html"
-COPYRIGHT="2003-2017 VideoLan.org"
+COPYRIGHT="2003-2019 x264 project"
```
Comment (waddlesplash) [inline media-libs/x264/x264-20170521.recipe]: Is this really correct?

Comment (kenmays) [reply media-libs/x264/x264-20170521.recipe]: Fixed.

**Comment by kenmays** [comment]
Only handbrake package affected on x86. No regression.

**Comment by kenmays** [comment]
Removing. May push for current version nearer R1B2 release review.

**Pull request closed without merging**